README.md

```diff
@@ -117,7 +117,12 @@ For a pagination style wrap your buttons in `.o-buttons-pagination`. In this exa
 ```
 
 ### Disabled
-`o-buttons` styles buttons with the `disabled` attribute to show they are inactive. In some cases it is preferred to visually hide the button until it is active. For this case add the class `o-buttons--hide-disabled`.
+
+Styles for disabled states are included in `o-buttons` however the use of disabled buttons is not recommended. Firstly, disabled buttons pose accessibility issues including being difficult to see. Although disabled buttons with low contrast are allowed according to [Web Content Accessibility Guidelines](https://www.w3.org/TR/WCAG21/#abstract), blind or visually impaired users may miss vital context. Secondly, disabled buttons may be frustrating for the user if they are unsure why the button is disabled.
```

Comment:
```suggestion
Disabled buttons have [poor contrast which makes them difficult to read](). Disabled buttons [do not give feedback to a user why they are disabled]().

Avoid them unless user research shows they improve your interface.
```

I would move some of the more academic nuances into a reference section at the bottom or in the commit message itself, as we want to keep the content as minimal as possible to what the reader needs to understand.